react/features/invite/components/add-people-dialog/web/AddPeopleDialog.js

```diff
@@ -283,24 +283,42 @@ class AddPeopleDialog extends AbstractAddPeopleDialog<Props, State> {
      * search autocomplete.
      */
     _parseQueryResults(response = []) {
-        const { t } = this.props;
+        const { t, _dialOutEnabled } = this.props;
         const users = response.filter(item => item.type !== 'phone');
-        const userDisplayItems = users.map(user => {
-            return {
-                content: user.name,
-                elemBefore: <Avatar
-                    className = { 'avatar-small' }
```

Comment:
We used to have to classes 'avatar-small' and 'avatar-xsmall' which are 28x28 and 16x16, one is used in elemBefore and one int tag: {elemBefore....}, what is the difference? And why you switched to using just the smaller one?
Other than that everything seems fine.
And if we are sure and want to remove one of them and not use it, can you clean up css, please.

Reply:
@damencho Thanks for the catch! Fixed it. 

One is the avatar for the `tag` (the input where the completions are added) and the other one for the element per se (the list entry). The tag avatar is smaller and the tag for list entry has to be bigger.